libcloud/compute/drivers/ec2.py

```diff
@@ -1862,17 +1862,47 @@ def create_node(self, **kwargs):
                                               the operating systems command
                                               for system shutdown.
         :type       ex_terminate_on_shutdown: ``bool``
+
+        :keyword    ex_launch_template_name: Template to create the instance
+                                             from.
+        :type       ex_launch_template_name: ``str`` or ``None``
+
+        :keyword    ex_launch_template_id:   Template ID to create instance
+                                             from.
+        :type       ex_launch_template_id:   ``str`` or ``None``
+
+        :keyword    ex_launch_template_version: Template version to create
+                                                instance from.
+        :type       ex_launch_template_version: ``str``, ``int`` or ``None``
         """
-        image = kwargs["image"]
-        size = kwargs["size"]
+
         params = {
             'Action': 'RunInstances',
-            'ImageId': image.id,
             'MinCount': str(kwargs.get('ex_mincount', '1')),
             'MaxCount': str(kwargs.get('ex_maxcount', '1')),
-            'InstanceType': size.id
         }
 
+        if 'ex_launch_template_name' in kwargs:
```

Comment:
How do those arguments interact with each other? Are they mutually exclusive aka only a single one can be provided or you can mix them and provide more than one?